orangecontrib/network/widgets/graphview.py

```diff
@@ -26,8 +26,7 @@ class QGraphicsEdge(QGraphicsLineItem):
     def __init__(self, source, dest, view=None):
         super().__init__()
         self.setAcceptedMouseButtons(Qt.NoButton)
-        self.setFlags(self.ItemIgnoresTransformations |
-                      self.ItemIgnoresParentOpacity)
+        self.setFlags(self.ItemIgnoresParentOpacity)
```

Comment:
Without `ItemIgnoresTransformations` the edges now scale with zoom, but, that was the only way I got it to work. 😊